patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
+index 72dde06a99a77b34f466cbfba3d478dc60b2d86c..293ded5d7615b4cc65fb739f7cde499ee39bf861 100644
+--- a/chrome/browser/ui/BUILD.gn
++++ b/chrome/browser/ui/BUILD.gn
+@@ -2669,6 +2669,12 @@ split_static_library("ui") {
+         "webui/settings/incompatible_applications_handler_win.h",
+       ]
+       deps += [ "//google_update" ]
++    } else if (brave_chromium_build && is_official_build) {
```

Comment:
can't we just add these in our own gn file?

Reply:
like you did here https://github.com/brave/brave-core/pull/109/files#diff-84f8700538180eec90bb2f4c9dff4da4R42